odfuzz/generators.py

```diff
@@ -88,10 +88,14 @@ def generate(generator_format='uri'):
 class EdmDecimal:
     @staticmethod
     def generate(self,generator_format='uri'):
-        divider = random.randint(1, 10 ** self.scale)
-        scale_range = random.randint(0, self.scale)
-        rand_int = random.randint(1, (10 ** (self.precision - scale_range)) - 1)
-        value = '{0:.{1}f}'.format(rand_int / divider, scale_range) + 'm'
+        if self.precision == self.scale:
```

Comment:
I guess it is OK, but if there were all generator.py test passing with this bug, then there is one test case definitelly missing

Reply:
addressed